btap/src/feature_selection.py

```diff
@@ -37,7 +40,8 @@ def select_features(args, estimator_type='lasso', min_features=20):
     data = acm.access_minio(operation='read',
                             path=args.in_obj_name,
                             data='')
-
+    logger.info("read from mino  ", data)
```

Comment:
The log statement has no placeholder for the data. Also, this will dump the whole dataframe into the log.

```suggestion
    logger.info("read from mino %s", data.head())
```